channels/tsmf/client/tsmf_media.c

```diff
@@ -882,7 +1026,7 @@ void tsmf_presentation_set_audio_device(TSMF_PRESENTATION* presentation, const c
 	presentation->audio_device = device;
 }
 
-static void tsmf_stream_flush(TSMF_STREAM* stream)
+void tsmf_stream_flush(TSMF_STREAM* stream)
```

Comment:
Yes, I am now calling this function from tsmf_ifman.c t support flush message per stream instead of for the entire presentation.